docs/rfc/block-registration.md

```diff
@@ -69,19 +69,32 @@ To register a new block type, start by creating a `block.json` file. This file:
 	"icon": "star",
 	"description": "Shows warning, error or success notices  ...",
 	"keywords": [ "alert", "message" ],
+	"textDomain": "my-plugin",
 	"attributes": {
 		"message": {
 			"type": "string",
 			"source": "html",
 			"selector": ".meessage"
 		}
 	},
-	"edit": "blocks/notice-edit.js",
-	"save": "blocks/notice-save.js",
 	"styles": [ 
 		{ "name": "default", "label": "Default", "isDefault": true }, 
 		{ "name": "other", "label": "Other" }
-	]
+	],
+	"editorScript": "file:build/editor.assets.json",
+	"script": {
+		"handle": "my-plugin-notice",
```

Comment:
Do we really need a "handle"? can't it be just auto-generated in PHP based on the script?

Reply:
Yep, I think you can generate them from the name of the block. This is how I did it myself :)